### PR TITLE
fix tests on old API

### PR DESCRIPTION
**Pull request by greenEkatherine**
related to #83

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -41,6 +41,9 @@ jobs:
 
             $ANDROID_HOME/platform-tools/adb devices
 
+            sed -i 's/^\(hw.sdCard=\).*$/\1yes/' ~/.android/avd/xamarin_android_emulator.avd/config.ini
```
Comment (omen273) [inline azure-pipelines.yml:44]: Please remove if you can

File: azure-pipelines.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
             echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-${{ value }};google_apis;x86'
             # -d 9 for Nexus 5X
             # to find out other devices run "avdmanager list device"
-            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' -d 9 --force
+            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' --sdcard 512M -d 9 --force
```
Comment (omen273) [inline azure-pipelines.yml:34]: Please add comment about why we need 512

**Review by omen273 — CHANGES_REQUESTED** [review]
Please address my comments

File: azure-pipelines.yml (hunk 2 above)
Comment (greenEkatherine) [reply azure-pipelines.yml]: Removed

Comment (greenEkatherine) [reply azure-pipelines.yml]: added

File: azure-pipelines.yml (hunk 1 above)
Comment (greenEkatherine) [reply azure-pipelines.yml]: removed

**Review by omen273 — APPROVED** [review]

**Pull request merged**